client/signup/steps/theme-selection/index.jsx

```diff
@@ -38,16 +38,19 @@ module.exports = React.createClass( {
 	},
 
 	pickTheme( theme ) {
-		const themeSlug = theme.id;
+		const repoSlug = `${ theme.repo }/${ theme.slug }`;
 
-		analytics.tracks.recordEvent( 'calypso_signup_theme_select', { theme: themeSlug, headstart: true } );
+		analytics.tracks.recordEvent( 'calypso_signup_theme_select', {
+			theme: repoSlug,
+			headstart: true
+		} );
 
 		SignupActions.submitSignupStep( {
 			stepName: this.props.stepName,
 			processingMessage: this.translate( 'Adding your theme' ),
-			themeSlug
+			repoSlug
```

Comment:
This is the change I was talking about. Does this affect this line? https://github.com/Automattic/wp-calypso/blob/c5ec5ae2eb110663b89379b19620e5fa0fdeda8a/client/lib/signup/step-actions.js#L22